starter-code/css/main.css

```diff
@@ -0,0 +1,75 @@
+h1 a{
+	color: #ffb782;
+	letter-spacing: 2px;
+	font-weight: 500;
+	text-decoration: none;
+	font-size: 48px;
+	
+}
+
+body {

```

Comment:
Since this applied to everything in the html, you want this to be at the top of your CSS file. Styles cascade down so you typically want the least specific selector at the top of the file, and the most specific at the bottom.